src/config/dynamic-theme-fixes.config

```diff
@@ -11037,6 +11037,21 @@ INVERT
 
 ================================
 
+rain.thecomicseries.com
+
+CSS
+html {
+    background-color: revert !important;
+}
```

Comment:
We shouldn't remove the background-color of `html` - especially not with `revert`. The background-color of html is used for initial rendering, removing the background-color will make the initial coloring white, which causes problems like white flashes/

Reply:
Ok, I can see how this does make sense.

The alternative fix would be to explicitly set the margin for the `body` tag to `0px`, which would remove the outline area. To keep the content layout looking the same, it should add padding equal to the margin being removed. I'll update the PR with this.